### PR TITLE
update DayPicker story to show single month

### PR DESCRIPTION
**Pull request by dburles**
Looks like at one point the default was changed to two months while the story was representing a default of one

### CODE REVIEW

File: stories/DayPicker.js [hunk 1]
```diff
@@ -52,12 +52,11 @@ storiesOf('DayPicker', module)
   .addWithInfo('with custom day size', () => (
     <DayPicker daySize={50} />
   ))
-  .addWithInfo('more than one month', () => (
```
Comment (ljharb) [inline stories/DayPicker.js]: We should have all of explicit 1, > 1, and "default". That way we don't have to update this in the future if the default changes.

File: stories/DayPicker.js [hunk 2]
```diff
@@ -52,12 +52,11 @@ storiesOf('DayPicker', module)
...
-    <DayPicker numberOfMonths={2} />
+  .addWithInfo('single month', () => (
+    <DayPicker numberOfMonths={1} />
   ))
   .addWithInfo('vertical', () => (
     <DayPicker
-      numberOfMonths={2}
```
Comment (ljharb) [inline stories/DayPicker.js]: Similarly, having these explicit ensures that a change of the default won't suddenly make these stories useless.

File: stories/DayPicker.js (hunk 1 above)
Comment (dburles) [inline stories/DayPicker.js]: Just noticed there's  a pattern used in other stories that's *single month* and *3 months*, I'll go with that

Comment (ljharb) [inline stories/DayPicker.js]: Sounds great!

**Comment by majapw** [comment]
Thanks!

**Pull request merged**